src/main/java/bowling/domain/BowlingGames.java

```diff
@@ -0,0 +1,64 @@
+package bowling.domain;
+
+import java.util.Collections;
+import java.util.List;
+
+public class BowlingGames {
+
+    private final int LAST_PLAYER_TURN_INDEX;
```

Comment:
인스턴스 필드는 자바 컨벤션으로 Lower Camel Case 를 활용해주시는건 어떨까요?
```suggestion
    private final int lastPlayerTurnIndex;
```